docs-v2/content/contributing/setting-up-a-dev-environment.md

```diff
@@ -30,35 +28,39 @@ If you run into trouble here, make sure your node, npm, and **_yarn_** are on th
 This method is slow, and if you are reading this tutorial you probably don't want to do it
 
 1.  `yarn bootstrap`
-2.  At the prompt: `a`, `enter`, `y`, `enter`
+2.  At the prompt: `a`, `enter`, `y`, `enter` ( `a` selects all (fully bootstrap) and `y` is to confirm)
 3.  Have a beer 🍺
-4.  `yarn test`
+4.  `yarn test` (to verify everything worked)
 
-### 2. Create a project (skip this if you already have a react project)
+### Working with the kitchen sink app
 
-1.  `cd ~` (optional)
-2.  `create-react-app storybook-sandbox-app`
-3.  `cd storybook-sandbox-app`
-4.  `yarn start`
-5.  Verify that you have a working react app.
+TODO: Fill this in
 
-### 3. Setup storybook in your project
-
-First we are going to install storyboook, then we are going to link `@storybook/react` into our project. This will replace `node_modules/@storybook/react` with a symlink to our local version of storybook. 
+## Connecting Your App To Storybook
 
-1.  `getstorybook`
-2.  `yarn storybook`
-3.  Verify that storybook works
+**_Note:_** If you aren't seeing addons after linking storybook, you probably have a versioning issue which can be fixed by simply linking each addon you want to use.
 
-### 4. Link storybook
+### Linking Storybook
 
-Storybook is broken up into sub-projects that you can install as you need them. For this experiment we will be working with `@storybook/react`. 
+Storybook is broken up into sub-projects that you can install as you need them. For this example we will be working with `@storybook/react`.
```

Comment:
Not a note on this PR, but we should totally make a CLI for this!!

Reply:
+1